### DISCUSSION TITLE
Windows start daemon only (no GUI) at startup

### DISCUSSION THREAD

**Discussion by Gornius** [OpenTabletDriver/OpenTabletDriver#1584] [Ideas]
So, there is better way (imo) to start driver at Windows startup. This way nothing pops up at start and no GUI is loaded (only daemon starts). If you want to reconfigure tablet you're free to launch GUI App.

1. Open `WIN+R`.
2. Launch `shell:startup`
3. Create new file `OTD.vbs`
4. Paste following code:
```
Set oShell = CreateObject ("Wscript.Shell") 
Dim OTDDaemonPath
OTDDaemonPath = "X:\Your\Path\To\OpenTabletDriver.win-x64\OpenTabletDriver.Daemon.exe"
oShell.Run OTDDaemonPath, 0, false
```
5. Don't forget to change `OTDDaemonPath` to your path to `OpenTabletDriver.Daemon.exe`.
6. You're good to go.

**Comment by jamesbt365** [comment]
we may plan on adding something like this in the future so all working around doesn't have to happen but how does attaching a ux to it work?

**Reply by Gornius** [reply]
If daemon is launched, it doesn't load its own daemon but attaches to it. You can close GUI and it still works.

**Comment by IdmiDwebe** [comment]
Thank you very much for this

To those who want to run OpenTabletDriver without GUI on Windows startup and with administrator privillages, you have to create a task in Task Scheduler and enable "run with highest privillages". Set it to run on login and then set the action to be to run "C:\System32\cscript.exe" and set the argument to be the location to the OpenTabletDriver.vbs script created here

**Reply by jamesbt365** [reply]
We don't support running otd as admin so any issue is on you and not us.

**Comment by PillTime** [comment]
The path should be triple quoted (`OTDDaemonPath = """X:\Your\Path\..."""`) to allow for paths with spaces.
